File: Data Science/Assignment 2.py
import pandas as pd

# Load dataset
url = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"

df = pd.read_csv(url)

# Create a new column
df["Average Score"] = (
    df["sepal_length"] +
    df["sepal_width"] +
    df["petal_length"] +
    df["petal_width"]
) / 4

# Apply a condition
df["Category"] = df["Average Score"].apply(
    lambda x: "High" if x >= 3.5 else "Low"
)

print(df.head())

File: Data Science R Language/assignment2.R
# Load dataset
url <- "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv"
df <- read.csv(url)

# Create Average Score column
df$Average_Score <- (
    df$sepal_length + df$sepal_width +
    df$petal_length + df$petal_width
) / 4

# Apply condition
df$Category <- ifelse(df$Average_Score >= 3.5, "High", "Low")

# Display first 5 rows
print(head(df))
